src/ripple/protocol/STAmount.h

```diff
@@ -104,6 +104,32 @@ class STAmount final
 
     STAmount (Issue const& issue, int mantissa, int exponent = 0);
 
+    std::size_t
+    size_of() const override
+    {
+        return sizeof(*this);
+    }
+
+    STBase*
+    copy (std::size_t n, void* buf) const override
```

Comment:
lol.  Ok.  I keep updating this little snippet with updated tweaks.  The last one was to make `emplace` static.  So you could pull `emplace` out of `STAmount` and plop it down in a `detail` namespace.